social-wall/index.html

```diff
@@ -21,25 +24,44 @@
   <body>
     <div id="search">
         <div class="input-group input-group-lg" style="float:left">
+<<<<<<< HEAD
+          <input name="q" id="query" type="text"
+                 class="center-block form-control input-lg"
+                 title="Search Twitter and the loklak network"
+                 placeholder="Search Twitter and the loklak network">
+          <span class="input-group-btn">
+          <button class="btn btn-lg btn-primary"
+                  id="search-but" type="button">
+          Search Twitter
+          </button>
+          </span>
+=======
           <input name="q" id="query" type="text" class="center-block form-control input-lg" title="Search Twitter and the loklak network" placeholder="Search Twitter and the loklak network">
```

Comment:
Line is longer than allowed. (190 > 129)

*LineLengthBear, severity NORMAL, section `default`.*